### DISCUSSION TITLE
Full Node will not start after Upgrade to v2.1.0

### DISCUSSION THREAD

**Discussion by k2skaterii** [guydavis/machinaris#933] [Q&A]
After upgrading to v2.1.0 machinaris appears to start up just fine, and some of the Chia services appear to start but the Full_node service does not start.  When I launch the Chia GUI from my laptop (Win11) it just sits there "starting services" and when I look at the Machinaris index page says "Farming: Not available".  

This is what I found while looking into the status of the Chia services 

```
root@helium-etl:/chia-blockchain# chia show -s
Error: Connection error: ClientConnectorError: Cannot connect to host 0.0.0.0:8555 ssl:<ssl.SSLContext object at 0x7f3676d59cc0> [Connect call failed ('0.0.0.0', 8555)]
Check if full node rpc is running at 8555
This is normal if full node is still starting up

```
```
root@helium-etl:/chia-blockchain# chia rpc status
╭──────────────┬───────────╮
│ SERVICE      │ STATUS    │
├──────────────┼───────────┤
│ crawler      │ INACTIVE  │
├──────────────┼───────────┤
│ daemon       │ ACTIVE    │
├──────────────┼───────────┤
│ farmer       │ ACTIVE    │
├──────────────┼───────────┤
│ full_node    │ INACTIVE  │
├──────────────┼───────────┤
│ harvester    │ ACTIVE    │
├──────────────┼───────────┤
│ timelord     │ INACTIVE  │
├──────────────┼───────────┤
│ wallet       │ ACTIVE    │
├──────────────┼───────────┤
│ data_layer   │ INACTIVE  │
╰──────────────┴───────────╯

```

While troubleshooting I updated my docker-compose.yaml to force "v2.0.0" instead of "latest", when I did that everything came back up like normal.  

My initial investigation is pointing me toward an SSL problem, but that doesn't really fit since the SSL problem seems to go away when I roll back.  Here's a sample of what's got me scratching my head.

```
2023-10-06T21:35:13.137 wallet wallet_server              : INFO     Cannot connect to host 127.0.0.1:8444 ssl:<ssl.SSLContext object at 0x7fcefa1e45c0> [Connect call failed ('127.0.0.1', 8444)]
2023-10-06T21:35:13.138 wallet wallet                     : INFO     Failed to connect to PeerInfo(_ip=IPv4Address('127.0.0.1'), _port=8444)
2023-10-06T21:35:14.056 farmer farmer_server              : INFO     Cannot connect to host 127.0.0.1:8444 ssl:<ssl.SSLContext object at 0x7fa1ea6cfec0> [Connect call failed ('127.0.0.1', 8444)]
2023-10-06T21:35:14.057 farmer farmer                     : INFO     Failed to connect to PeerInfo(_ip=IPv4Address('127.0.0.1'), _port=8444)

```

Now, I always blame the dummy behind the keyboard first.  So is there something I could have done wrong here?  Or something that I should have done before upgrading that I didn't do because I didn't RTFM?  

I've rolled back to v2.0.0 for now, but if anyone has any ideas about what could be going on please lemme know!  If there are logs or config files I can provide to help troubleshoot just let me know.  

Thank you!

**Comment by guydavis** [comment]
Sorry to hear that. Other users upgraded without incident to Machinaris 2.1.0, so I think there may be something in your local install/config.  Please try a [full blockchain config reset](https://github.com/guydavis/machinaris/wiki/Troubleshooting#reset-blockchain-config) when you update next.

**Comment by k2skaterii** [comment]
Resetting the Blockchain config didn't work... But Resetting the Blockchain DB did.  I'm just glad we have the option to download a "Database Checkpoint" from https://www.chia.net/downloads/.  

I'm back up and running now!

**Reply by guydavis** [reply]
Great to hear.  Yes, the Chia DB is not particuarly reliable.  Yes, you can manually download a DB checkpoint from Chia or have Machinaris [download it automatically](https://github.com/guydavis/machinaris/wiki/Blockchains#blockchain-download), from same the source.

**Reply by k2skaterii** [reply]
There was a way to have machinaris do that for me!  I suppose not knowing that truly is a RTFM situation!  I can't really complain, this is the 1st time I've had to re-sync since I first deployed my Chia farm in June/July 2021!  

Seems I got my setup back online just in time for my favorite little picture to show up in my machinaris GUI!
![image](https://github.com/guydavis/machinaris/assets/87842648/688be564-f7b9-4530-880c-a9e98f7fbe6f)
